jboss-eap-clustered-rhel-VMSS/README.md

```diff
@@ -37,31 +37,31 @@ Following is the Architecture :
 To learn more about JBoss Enterprise Application Platform, check out:
 https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.2/
 
-## Licenses and Costs 
+## Licenses, Subscriptions and Costs 
 
-This uses RHEL 8.0 image which is a PAY AS YOU GO image and doesn't require the user to license it, it will be licensed automatically after the instance is launched first time and user will be charged hourly in addition to Microsoft's Linux VM rates.  Click [here](https://azure.microsoft.com/en-gb/pricing/details/virtual-machines/linux/#red-hat) for pricing details.
+This template uses a RHEL 8.0 image which is a Pay-as-you-go (PAYG) image and does not require the user to license.  The VM will be licensed automatically after the instance is launched first time and the user will be charged hourly in addition to Microsoft's Linux VM rates.  Click [here](https://azure.microsoft.com/en-gb/pricing/details/virtual-machines/linux/#red-hat) for pricing details.
```

Comment:
Since we have added the feature of selecting between offers (PAYG/BYOS), I have made the following changes to the template. Please review and suggest.

"If you select the offer as Pay-As-You-Go, template will deploy RHEL 8.0 Pay-As-You-Go image which carries a separate hourly charge that is in addition to Microsoft's Linux VM rates. In this case the VM will be licensed automatically after the instance is launched for the first time and total price of the VM consists of the base Linux VM price plus RHEL VM image surcharge. See Red Hat Enterprise Linux pricing for details. You also need to have a Red Hat account to register to Red Hat Subscription Manager (RHSM) and install JBoss EAP. To use the Enterprise Application Platform your RHSM account needs EAP entitlement. You can get an evaluation account for EAP from here. If you select the Offer as BYOS for deploying the template, your RHSM account must have both Red Hat Enterprise Linux entitlement (for subscribing the RHEL OS for the VM) and EAP entitlement and you will have to enter 2 pool IDs as mentioned in the template. Click here to know more about RHSM and pricing."